torch/quantization/fx/prepare.py

```diff
@@ -458,7 +464,23 @@ def maybe_insert_output_observer_for_node(
                     get_default_output_activation_post_process_map().get(
                         matched_pattern,
                         act_post_process_ctr)
+
+            # Find the next node in the graph
+            next_node = node.next
```

Comment:
this seems unintuitive, is there a design document which explains what we are trying to do?